io.github.baldeuniversel.am-okay.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+
+  <id>io.github.baldeuniversel.am-okay</id>
+  
+  <name>am-okay</name>
+  <summary>The `am-okay` program allows dynamically to copy/move files/directories (via the terminal)</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  
+  <description>
+    <p>
+      The <code>am-okay</code> program allows to copy or move files or directories in a dynamic mode via a console (terminal).
+    </p>
+    <p>
+      With the <code>am-okay</code> program, you can dynamically copy or move files or directories where access is permitted. The program itself partly imitates the Linux <code>cp</code> , <code>mv</code> commands but dynamically. You can tag files or  directories as items to copy or move via a console (terminal), and paste these items via another  console (terminal).
+    </p>
+    <p>
+      A lot of actions are possible, you can even tag files or directories in arrays and paste them  wherever you want in your Linux system.
+    </p>
+  </description>
+  
+  <icon type="stock">org.flatpak.am-okay</icon>
```

Comment:
```suggestion
```

stock icons don't work

you should (and are missing) have a launchable instead to point at the (currently) missing .desktop file